common/djangoapps/enrollment/tests/test_data.py

```diff
@@ -378,3 +378,21 @@ def assert_enrollment_modes(self, expected_modes, include_expired):
 
         if not include_expired:
             self.assertNotIn('verified', result_slugs)
+
+    def test_get_roles(self):
+        """Create a role for a user, then get it"""
+        expected_role = CourseAccessRoleFactory.create(course_id=self.course.id, user=self.user, role="SuperCoolTestRole")
+        expected_roles = set()
```

Comment:
Note, you can create a new set with members in by doing `expected_roles = {expected_role}`.